components/IscnUploadForm.vue

```diff
@@ -837,6 +913,11 @@ export default class IscnUploadForm extends Vue {
 
     try {
       this.uploadStatus = 'uploading';
+
+      if (this.checkUploadFileTypeIsPDF()) {
```

Comment:
It seems the function does not simply check `is PDF`, but `PDF and not EPUB`, may be split 2 logic?
Sth like `this.checkUploadFileTypeContainsPDF() && !this.checkUploadFileTypeContainsEPUB()` (actually simple `.find` would do, we might not even need a function`

Reply:
@AuroraHuang22
[🎨 Apply simplification suggestion](https://github.com/likecoin/app-like-co/pull/457/commits/79cd883bbdabd207ea9772326f58a5c90d271592)